### ISSUE TITLE
_index.md in Hugo - it is still showing up and editable 

### ISSUE THREAD

**Issue by sardarms** [sveltia/sveltia-cms#210]
Not sure if this is a bug, but I can still see and edit the _index.md file in my hugo site. 

Here is my code
```
collections:
  
  - name: 'posts-en'
    label: 'Posts-En'
    folder: 'content/en/blog'
    create: true
    slug: '{{slug}}'
    editor:
      preview: true
    fields:
      - { label: 'Title', name: 'title', widget: 'string' }
      - { label: 'Publish Date', name: 'date', widget: 'datetime', dateFormat: 'YYYY-MM-DD', format: 'YYYY-MM-DD', required: true }
      - { label: "Featured image", name: "feature_image", widget: "image", required: false }
      - { label: "Description for SEO", name: "description", widget: "string", required: false }
      - { label: "Categories", name: "categories", widget: "list", required: false, field: { name: text, label: Text, widget: string } }
      - { label: "Tags", name: "tags", widget: "list", required: false, field: { name: text, label: Text, widget: string } }
      - { label: "Author", name: "author", widget: "string", required: true }
      - { label: "Body", name: "body", widget: "markdown" }
``` 

In the blog folder, I have _index.md file and some other post. But in the editor, I can still access this and edit the content of the _index.md file.

**Comment by kyoshino** [comment]
Will check.

**Comment by kyoshino** [comment]
Hmm, I cannot reproduce the issue. Could you share your entire config file? If it’s not suitable to paste here, you can [email it to me](mailto:kohei@britegrid.io) instead.

**Comment by kyoshino** [comment]
I could reproduce the issue with the GitHub backend but not locally. Figuring out why.

**Comment by kyoshino** [comment]
Fixed in [v0.39.15](https://github.com/sveltia/sveltia-cms/releases/tag/v0.39.15)!

**Comment by sardarms** [comment]
It's gone now. 🙏
